publish.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# This script can be executed to publish all results found in `result` subdir. It will
+# compress the loader image because Capstan requires compressed file when pulling 
+# images from the repository. Then it will look for all package files and their manifests
+# (*.mpm and *.yaml) and upload them sequentially.
+#
+# Before using this script, you have to configure s3cmd (http://s3tools.org/s3cmd).
+# After installing, you can invoke the configuration command
+#
+# $ s3cmd --configure
+#
+# You will be asked few AWS related questions (region, key, secret) that you need to
+# provide for the user with write access.
+
+pushd result
+
+echo "Compressing OSv loader image"
+gzip osv-loader.qemu
+echo "Uploading OSv loader image"
+s3cmd put --acl-public --guess-mime-type osv-loader.qemu.gz s3://mikelangelo-capstan/mike/osv-loader/osv-loader.qemu.gz
+
+FILES=`ls -1 *.{yaml,mpm}`
```

Comment:
Would it be difficult to produce the results in a more "repository-like" structure? On S3 we have the following structure

```
mike/osv-loader/...
packages
```

Since we can have multiple osv-loaders, I was thinking if you could create the same structure in your `result` dir. This way, the script will simply upload whatever it finds in each dir.